homework/index.js

```diff
@@ -1,46 +1,165 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
+  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  // defined for taking data from url
+  function fetchJSON(url, callBackFunction) {
     const xhr = new XMLHttpRequest();
     xhr.open('GET', url);
     xhr.responseType = 'json';
     xhr.onload = () => {
       if (xhr.status >= 200 && xhr.status <= 299) {
-        cb(null, xhr.response);
+        callBackFunction(null, xhr.response);
       } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+        callBackFunction(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
       }
     };
-    xhr.onerror = () => cb(new Error('Network request failed'));
+    xhr.onerror = () => callBackFunction(new Error('Network request failed'));
     xhr.send();
   }
-
+  // defined for new element creat&append
```

Comment:
1. What do these comments add what is not already indicated by the function name? If you think the function name is not clear enough to describe what the function does, it is better to change the function name rather than using a comment to fix an inadequate name. Use comments as your last resort.

Even with a comment, separate function declarations with a blank line, just as you would separate paragraphs in a piece of text with a blank line for readability.